ui/app/src/modules/Content/Authoring/NewContentDialog.tsx

```diff
@@ -44,6 +43,11 @@ import { SuspenseWithEmptyState } from '../../../components/SystemStatus/Suspenc
 import { LegacyFormConfig } from '../../../models/ContentType';
 import { Resource } from '../../../models/Resource';
 import StandardAction from '../../../models/StandardAction';
+import { useDispatch } from 'react-redux';
+import SingleItemSelector from './SingleItemSelector';
+import { SandboxItem } from '../../../models/Item';
+import { showErrorDialog } from '../../../state/reducers/dialogs/error';
+import { Dialog } from '@material-ui/core';
```

Comment:
Make sure to import the dialog from its file and not from core (check all over the dialogs where you did the find & replace)